### DISCUSSION TITLE
Regarding TEI huggingface docs

### DISCUSSION THREAD

**Discussion by SMAntony** [huggingface/text-embeddings-inference#323] [Ideas]
Is there any plan to update the TEI docs?

Compared to TGI docs, TEI docs feels a little rough. I came across one inconsistency and I suspect there to be more.

Inconsistency:
In [quicktour](https://huggingface.co/docs/text-embeddings-inference/en/quick_tour), we are told that safetensors for ```BAAI/bge-large-en-v1.5``` is still a pr but its has been merged.

If I am not wrong here and if anyone is interested in guiding me. I will be happy to raise a pr for docs change.

**Comment by OlivierDehaene** [comment]
Hello! Feel free to submit a PR if you find inconsistencies.
